### PR TITLE
DDNS-Updater: Add name.com provider

### PR DESCRIPTION
**Pull request by reederda**

### CODE REVIEW

**Review by stavros-k — CHANGES_REQUESTED** [review]
Hello, please make changes only under `library/ix-dev/community/ddns-updater`
The directory you touched now is autogenerated.

Thanks

**Comment by reederda** [comment]
Apologies. I've made the changes to the correct directory.

**Review by stavros-k — CHANGES_REQUESTED** [review]
Please revert changes on the `community/ddns-updater/1.0.16` directory.

File: library/ix-dev/community/ddns-updater/ci/name-values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      nameUsername: "abcdef123456abcdef123456abcdef12"
+      nameToken: "abcdef123456abcdef123456abcdef12"
+      nameTTL: 300
+    - provider: name
+      host: subdomain
+      domain: example.com
+      ipVersion: ""
+      nameUsername: "abcdef123456abcdef123456abcdef12"
+      nameToken: "abcdef123456abcdef123456abcdef12"
+      nameTTL: 300
```
Comment (stavros-k) [inline library/ix-dev/community/ddns-updater/ci/name-values.yaml:27]:
```suggestion
      nameTTL: 300

```

File: library/ix-dev/community/ddns-updater/templates/dnsProviders/_name.tpl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{{/*
+ddnsConfig:
+  config:
+    - provider: name                    - Required
+      domain: "example.com"             - Required
+      host: "@"                         - Required - Valid value ("@" or subdomain)
+      ipVersion: ""                     - Required - Valid values (ipv4/ipv6/"")
+      nameToken: token                  - Required
+      nameUsername: username            - Required
+*/}}
```
Comment (stavros-k) [inline library/ix-dev/community/ddns-updater/templates/dnsProviders/_name.tpl:16]:
```suggestion
*/}}

```

**Pull request closed without merging**